NuGet.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It configures the NuGet feeds used by DNX/DNU. Makes it easier to swap out a certain feeds, rather than using the global NuGet.config.

Other repo's use it as well.